packages/contracts-bedrock/scripts/DeploySuperchain.s.sol

```diff
@@ -377,7 +374,7 @@ contract DeploySuperchain is Script {
 
     // -------- Utilities --------
 
-    function etchIOContracts() internal returns (DeploySuperchainInput dsi_, DeploySuperchainOutput dso_) {
+    function etchIOContracts() public returns (DeploySuperchainInput dsi_, DeploySuperchainOutput dso_) {
```

Comment:
Personally, I wouldn't intuitively know to call `etchIOContracts` before I was able to call `run(DeploySuperchainInput _dsi, DeploySuperchainOutput _dso)`. 

Unless we eventually have docs on how to interact with each script.